propolis-server-lib/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "propolis-server-lib"
+version = "0.1.0"
+authors = ["Sean Klein <sean@oxide.computer>", "Patrick Mooney <pmooney@oxide.computer>"]
+license = "MPL-2.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0"
+bitflags = "1.2"
+# dropshot = "0.6"
+dropshot = { git = "https://github.com/oxidecomputer/dropshot", branch = "main" }
+futures = "0.3"
+libc = "0.2"
+thiserror = "1.0"
+tokio = { version = "1", features = ["full"] }
+toml = "0.5"
+serde = "1.0"
+serde_derive = "1.0"
+schemars = { version = "0.8", features = [ "uuid" ] }
+structopt = { version = "0.3", default-features = false }
```

Comment:
You don't need this in propolis-server-lib as far as I can tell

Reply:
Good catch; removing this and a handful of other unnecessary deps.